tmux.1

```diff
@@ -5567,24 +5567,27 @@ The following keys have a special meaning in the command prompt, depending
 on the value of the
 .Ic status-keys
 option:
-.Bl -column "FunctionXXXXXXXXXXXXXXXXXXXXXXXXX" "viXXXX" "emacsX" -offset indent
-.It Sy "Function" Ta Sy "vi" Ta Sy "emacs"
-.It Li "Cancel command prompt" Ta "q" Ta "Escape"
-.It Li "Delete from cursor to start of word" Ta "" Ta "C-w"
-.It Li "Delete entire command" Ta "d" Ta "C-u"
-.It Li "Delete from cursor to end" Ta "D" Ta "C-k"
-.It Li "Execute command" Ta "Enter" Ta "Enter"
-.It Li "Get next command from history" Ta "" Ta "Down"
-.It Li "Get previous command from history" Ta "" Ta "Up"
-.It Li "Insert top paste buffer" Ta "p" Ta "C-y"
-.It Li "Look for completions" Ta "Tab" Ta "Tab"
-.It Li "Move cursor left" Ta "h" Ta "Left"
-.It Li "Move cursor right" Ta "l" Ta "Right"
-.It Li "Move cursor to end" Ta "$" Ta "C-e"
-.It Li "Move cursor to next word" Ta "w" Ta "M-f"
-.It Li "Move cursor to previous word" Ta "b" Ta "M-b"
-.It Li "Move cursor to start" Ta "0" Ta "C-a"
-.It Li "Transpose characters" Ta "" Ta "C-t"
+.Bl -column "FunctionXXXXXXXXXXXXXXXXXXXXXXXXX" "viXXXXX" "viXXXXX" "emacsX"
```

Comment:
I removed the offset in order to squeeze in the vi normal (abbreviated to `nml`) column and still make the this table fit into an 80 character wide terminal.